src/Components/FirstScreen/Net-component/style.scss

```diff
@@ -0,0 +1,47 @@
+.net-component {
+    position: fixed;
+    top: 0;
+    left: -52%;
+    width: 200%;
+    height: 130%;
+    overflow: hidden;
+    z-index: -1;
+
+    .net-svg {
+        position: absolute;
+        top: -10%;
+        left: -53%;
+        width: 210%;
+        height: 100%;
+
+
+        @media(max-width:1400px) {
```

Comment:
hmm, i would assume you will use these specific  screen breakpoints across whole app, right ? So, perhaps you can also  create variables for them ? https://css-tricks.com/approaches-media-queries-sass/